README.md

```diff
@@ -1,219 +1,219 @@
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="img/index.jpg" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Gergito Portfoilo Collaboration with teamate</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio-setup-and-mobile-first] <a name="about-project"></a>
+
+**[This port folio is for Gergito As a software developer]**
+
+## 🛠 Built With <a name="built-with"></a>
+-Html
+-Css
+-Github
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[added html, css, ]**
+- **[Github flow and linters]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+### Prerequisites
+
+1- Web browser
+2- Code editor
+3- Version Control System (git)
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/mohashyne/msa-portfolio.git
+```
+
+-
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with:
+
+Run this command:
+
+```sh
+  cd msa-portfolio
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+launch index.html using live server or live preview
+```
+
+### Run tests
+
+comming soon
+
+### Deployment
+
+You can deploy this project using:
+
+comming soon
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Gergito K.**
+
+- GitHub: [@githubhandle](https://github.com/gergitokusse)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)

```

Comment:
[Optional] It's a good practice to have the names of the contributors listed. I recommend that you update the list of contributors.

Reply:
: Wow I am sorry to forget them it also needed to acknowledge them on the acknowledgment part.
👍 👍 👍  Thank you again and again for an interesting comment. 👍 👍 👍